subsys/zigbee/osif/zb_nrf_platform.c

```diff
@@ -298,6 +298,10 @@ int zigbee_init(void)
 	 */
 	zb_set_nvram_erase_at_start(ZB_FALSE);
 
+#ifdef CONFIG_ZIGBEE_TC_REJOIN_ENABLED
```

Comment:
Is there a reason for this place to avoid using `S_ENABLED(CONFIG_ZIGBEE_TC_REJOIN_ENABLED)`?